api-1.11/src/main/java/org/openmrs/module/fhir/helper/ConditionHelperImpl1_11.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.helper;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.hl7.fhir.dstu3.model.Condition;
+import org.openmrs.Obs;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.fhir.api.helper.ConditionHelper;
+import org.openmrs.module.fhir.api.util.FHIRConditionUtil;
+import org.openmrs.module.fhir.api.util.FHIRConstants;
+import org.openmrs.module.fhir.api.util.FHIRUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component(value = "fhir.ConditionHelper")
+@OpenmrsProfile(openmrsVersion = "1.11.*")
+public class ConditionHelperImpl1_11 implements ConditionHelper {
+
+	/**
```

Comment:
Instead of repeating these methods so many times with the exact same implementation, can't we put this in some sort of base class?